rero_ils/modules/documents/dojson/contrib/jsontomarc21/model.py

```diff
@@ -582,53 +697,92 @@ def reverse_physical_description(self, key, value):
     return result or None
 
 
+@to_marc21.over('6XX', '^subjects')
+@utils.reverse_for_each_value
+@utils.ignore_value
+def reverse_subjects(self, key, value):
```

Comment:
reverse seems useless as it is a `to_marc21` transformation

Reply:
Every function in this `to_marc21` transformation is called `reverse_XXXX`.